### ISSUE TITLE
Subscribe to Bluesky Feeds

### ISSUE THREAD

**Issue by TomCasavant** [snarfed/bridgy-fed#1095]
This is probably out-of-scope, especially while the user-following is still being worked on, and I know the Group implementation of the ActivityPub spec is still a bit undefined particularly in microblog platforms, but does atProto treat Feeds as followable entities and could the bridge translate those into activitypub groups? 

I know the feed generators each have a unique `did` associated with them but I don't know if they have a username that could be easily translated into an AP username

What I'm picturing right now is I follow @columbusnews.feeds.bsky.social@bsky.brid.gy and then every post that ends up in that feed gets bridged over to my timeline

**Comment by snarfed** [comment]
Yes! Thanks for filing, great feature request. It'd be a project, including nontrivial work in product, UX, and eng. First step here would be to more fully flesh out the req'ts and UX. Your proposal is that Bluesky feed generators become fediverse bot users? I'd be curious to see the current complete set of workflows and interactions around Bluesky feed generators, and how all of those either map to fediverse actors well, or are unnecessary and we don't need to bridge.

**Comment by TomCasavant** [comment]
I don't know if them being bots is necessary, I was thinking more like how link aggregator (lemmy/kbin) communities are rendered in mastodon, e.g. a user account that essentially boosts the posts that are added to that community.

Though I guess that would look pretty much identical to how those @a.gup.pe (https://github.com/immers-space/guppe) work and I'm not sure if those are set up as bots or as actual Groups

**Comment by snarfed** [comment]
Right, at an AP level, "bot" users are basically the same as human users, the only difference is that they're AS2 `type=Service` or `Application` instead of `Person`. Behavior differences come from elsewhere.

**Comment by TomCasavant** [comment]
Oh sorry I thought you were referring to the 'bot' tag that mastodon uses. Yeah I was thinking just the entire feed becomes a bot user.

I believe I read somewhere when I was messing around with feeds a few weeks ago that a FeedGenerator essentially just boils down to a list of post urls/uris, so I don't know if you'd be able to get a stream of posts from them or if it'd have to routinely poll the feed generator and get the new set of posts every x minutes. But yeah, probably not important right now as I expect it's not as useful until significantly more bluesky accounts are opted in

**Comment by Tamschi** [comment]
This would definitely be tricky. feeds on Bluesky are exclusively polled-when-viewed, I think, and there are also shuffled feeds that would respond with a different random set of posts each time, which would likely lead to excessive POSTs on the ActivityPub side if implemented without safeguards against that (or missing posts if the safeguards are too strong).
